src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColorBlack;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $mainColorGray;
+    margin: 0 0 16px;
+  }
+
+  &__reviews-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $mainColorBlack;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 16px;
+  }
+
+  &__price-lable {
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColorGray;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $mainColorBlack;
+  }
+
+  &__buy-botton {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    background: $mainColorBlue;
+    border-radius: 5px;
+    font-weight: 700;
+    border: 1px solid transparent;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  &__buy-botton--active {
```

Comment:
You don't need separate class for button hover. Just nest " &:hover " into " &__buy-button ", and add transition for smooth hover

Reply:
Maybe you're right, but I think that in my version it is easier to reuse the card's block in the future if necessary.